src/DynamoCoreWpf/Views/Core/ConnectorPinView.xaml.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Forms;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Threading;
+using Dynamo.Configuration;
+using Dynamo.Controls;
+using Dynamo.Graph.Connectors;
+using Dynamo.Selection;
+using Dynamo.UI;
+using Dynamo.UI.Controls;
+using Dynamo.UI.Prompts;
+using Dynamo.Utilities;
+using Dynamo.ViewModels;
+using DynCmd = Dynamo.Models.DynamoModel;
+using MouseEventArgs = System.Windows.Input.MouseEventArgs;
+
+namespace Dynamo.Nodes
+{
+    /// <summary>
+    /// Interaction logic for ConnectorPinView.xaml
+    /// </summary>
+    public partial class ConnectorPinView : IViewModelView<ConnectorPinViewModel>
+    {
+        public ConnectorPinViewModel ViewModel { get; private set; }
```

Comment:
does this need to be public?

Reply:
It does bc the `IViewModelView` interface requires it. That interface doesn't do much but it might be nice to keep it for consistency?

Reply:
Fair enough